Project.toml

```
@@ -1,10 +1,14 @@
 name = "RustyObjectStore"
 uuid = "1b5eed3d-1f46-4baa-87f3-a4a892b23610"
-version = "0.8.2"
+version = "0.9.0"
```

```suggestion
version = "0.9.1"
```

just to match object_store_ffi_jll version